app/models/optout/NextUpdatesQuarterlyReportingContentChecks.scala

```diff
@@ -16,11 +16,16 @@
 
 package models.optout
 
-case class NextUpdatesQuarterlyReportingContentChecks(currentYearItsaStatus: Boolean, previousYearItsaStatus: Boolean, previousYearCrystallisedStatus: Option[Boolean]) {
+import models.itsaStatus.ITSAStatus.{Mandated, Voluntary}
+import services.optout.OptOutProposition
 
-  private def showOptOutContent: Option[Unit] = if (currentYearItsaStatus || (previousYearItsaStatus && !previousYearCrystallisedStatus.getOrElse(false))) Some({}) else None
+case class NextUpdatesQuarterlyReportingContentChecks(proposition: OptOutProposition) {
```

Comment:
this class can be named something like OptOutStatesViewModel, the important point is 'xxxViewModel' to indicate this will be passed to view page